firefox_relay_ToS/en-US.md

```diff
@@ -0,0 +1,60 @@
+
+## You Are Responsible For the Consequences of Your Use of the Service
+
+* You assure Mozilla that you will not use the Service to infringe anyone’s rights or violate any law.
+
+* To the extent permitted by applicable law, you agree that Mozilla will not be liable in any way for any inability to use the Service, for any limitations of the service, or for any claim arising out of these terms. Mozilla specifically disclaims the following: Indirect, special, incidental, consequential, or exemplary damages, direct or indirect damages for loss of goodwill, work stoppage, lost profits, loss of data, or computer malfunction. Any liability for Mozilla under this agreement is limited to $500.
+
+* You agree to indemnify and hold Mozilla harmless for any liability or claim that results from your participation in Firefox Relay, to the extent permitted by applicable law.
+
+* Mozilla provides the service “as is.” To the extent permitted by applicable law, Mozilla specifically disclaims any legal guarantees or warranties such as “merchantability,” “fitness for a particular purpose,” “non-infringement,” and warranties arising out of a course of dealing, usage, or trade.
+
+* You may have other rights under your own country’s laws. Nothing in these Terms of Service is intended to affect those rights, if they apply.
+
+## Updating or Ending These Terms
+
+* __Mozilla Can Update These Terms.__ Every once in a while, Mozilla may decide to update these Terms. We will post the updated Terms online. We will take your continued use of the Service as acceptance of such changes. We will post an effective date at the top of this page to make it clear when we made our most recent update.
+
+* __Termination.__ These Terms apply until either you or Mozilla decide to end them. You can choose to end them at any time for any reason by stopping your use of the Service and deleting the Firefox Relay Add-on. Mozilla can suspend or end anyone’s access to the Service at any time for any reason, including if Mozilla decides to end the Service. If we decide to suspend or end your access, we will try to notify you at the email address associated with your account or the next time you attempt to access your Account.
+
+  If we terminate your access to the Service without specific cause, you will continue to have access to the Service for the current billing period or we will refund the current month. However, we reserve the right to terminate your use of the Service immediately if we reasonably believe you have violated the Conditions of Use, or have otherwise violated these Terms.
+
+* __Choice of Law.__ California law applies to this contract, except for California’s conflict of law. If there is any conflict between this English version of the contract and a translation, this English version applies.
+
+
+### Contact Mozilla:
+
+Mozilla Corporation
+Attn: Mozilla – Legal Notices
+331 E. Evelyn Ave.,
+Mountain View, CA 94041
```

Comment:
In markdown, in order to force a line break, you need to add two space characters after each line. Otherwise I'll just turn this into a single line.